accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerManager.scala

```diff
@@ -36,30 +34,30 @@ class AccumuloLayerManager(attributeStore: AccumuloAttributeStore, instance: Acc
     AccumuloLayerDeleter(attributeStore, instance).delete(id)
 
   def copy[
-    K: AvroRecordCodec: Boundable: JsonFormat: ClassTag,
+    K: AvroRecordCodec: Boundable: Encoder: Decoder: ClassTag,
```

Comment:
I believe `Codec` can be used when both are expected

Reply:
(only going to make this comment here to avoid spamming the entire PR)

Reply:
@moradology there is no `Codec` type in circe AFAIK, can you point me to it?